src/launchpad/analyzers/android.py

```diff
@@ -19,6 +21,23 @@ def analyze(self, artifact: AndroidArtifact) -> AndroidAnalysisResults:
             package_name=manifest_dict["package_name"],
         )
 
+        apks: list[APK] = []
```

Comment:
Can this be pushed into artifacts? From the analyzer POV I don't see why it would concern itself with these details other than calling `artifact.get_primary_apks()`.

Reply:
Chatted offline, kicking this to future cleanup